packages/contracts/controller/src/account.cairo

```diff
@@ -147,6 +152,7 @@ mod CartridgeAccount {
 
     #[storage]
     struct Storage {
+        guardian_guid: Option<felt252>,
```

Comment:
wondering if this could be `guardian_guid: felt252` and the 0 value could be treated none?

Reply:
It could, less readable, but perhaps more efficient

Reply:
imo it would be a bit more readable since we do `unwrap_or(0);` so many places and essentially treat is as that

Reply:
I partially agree :)). All the unwraps happen in the session component, so the main account is more readable with an option. We could cache the sessions using (felt, Option<felt>, felt) instead of (felt, felt, felt), but we break compatibility with the sessions stored pre-guardian. 
wdyt? I'm not advocating for any of the two to be clear (felt everywhere, or optionable storage)

Reply:
i think lets roll with felt everywhere, it also avoids a storage slot

Reply:
ok!